suite-common/bluetooth/src/bluetoothReducer.ts

```diff
@@ -0,0 +1,150 @@
+import { AnyAction, Draft } from '@reduxjs/toolkit';
+
+import { createReducerWithExtraDeps } from '@suite-common/redux-utils';
+import { deviceActions } from '@suite-common/wallet-core';
+
+import {
+    bluetoothAdapterEventAction,
+    bluetoothConnectDeviceEventAction,
+    bluetoothKnownDevicesUpdateAction,
+    bluetoothNearbyDevicesUpdateAction,
+    bluetoothRemoveKnownDeviceAction,
+    bluetoothScanStatusAction,
+} from './bluetoothActions';
+
+export type BluetoothScanStatus = 'idle' | 'running' | 'error';
+
+export type DeviceBluetoothStatus =
+    | { type: 'pairing'; pin?: string }
+    | { type: 'paired' }
+    | { type: 'connecting' }
+    | { type: 'connected' }
+    | {
+          type: 'error';
+          error: string;
+      };
+
+// Do not export this outside of this suite-common package, Suite uses ist own type
+// from the '@trezor/transport-bluetooth' and mobile (native) have its own type as well.
+export type BluetoothDeviceCommon = {
+    id: string;
+    name: string;
+    data: number[]; // Todo: consider typed data-structure for this
+    lastUpdatedTimestamp: number;
+};
+
+export type DeviceBluetoothStatusType = DeviceBluetoothStatus['type'];
+
+export type BluetoothState<T extends BluetoothDeviceCommon> = {
+    adapterStatus: 'unknown' | 'enabled' | 'disabled';
+    scanStatus: BluetoothScanStatus;
+    nearbyDevices: Array<{ device: T; status: DeviceBluetoothStatus | null }>;
```

Comment:
Move `BluetoothDeviceState` above, then you can do this, just like in `initialState`
```suggestion
    nearbyDevices: BluetoothDeviceState<T>[];
```

Reply:
Good point, fixed (it was forgotten)

Reply:
https://github.com/trezor/trezor-suite/pull/17090/commits/20fcc715922b30b9b128e5c153fdee558c97dbc0